Manito/Manito/Screens/CreateRoom/Component/CheckRoomView.swift

```diff
@@ -10,6 +10,11 @@ import UIKit
 import SnapKit
 
 class CheckRoomView: UIView {
+    var dateRange = "" {
+        didSet {
+            dateLabel.text = dateRange
+        }
```

Comment:
p5;

```suggestion
        willSet {
            dateLabel.text = newValue
        }
```
이렇게 하면 원하는대로 구현이 안될까요?

Reply:
문제없이 사용 가능한데 혹시 didSet과 willSet을 구분해서 사용하는 큰 이유가 있을까요 ??
지금과 같이 명확히 한가지를 써야하는 상황이 아닌 경우엔 어떤걸 쓰는게 더 좋을까요 ??

Reply:
didSet, willSet를 사용하면 내부 값으로 들어오는 oldValue, newValue의 사용이 가능한데, dateRange를 가져와서 써야할까 하는 생각이 들어서 제안해봤습니당.

Reply:
아하아하 그렇군요 방금 적으면서 didSet의 oldValue에 대해 헷갈리고 있었다는 걸 알았습니다.
두 친구의 역할이 확실이 다르군요 !